### PR TITLE
Update Featured Project - Milky Way Project

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
* This PR changes the Featured Project on the Home Page.
* The new project is the [Milky Way Project](https://www.zooniverse.org/projects/povich/milky-way-project)
screen shot 2017-07-11 at 15 36 43
* **Note:** the image is taken from the background image of the project itself.
* **Note:** the image has no `alt` tag (unlike the previous FP). If this is an issue, please let me know of a suggested description. ( _"An image from the Milky Way galaxy"_ ?)

Tagging the original requester, @mrniaboc 

### Status
Ready for review.

### Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

### CODE REVIEW

File: app/pages/home-common/featured-project.jsx [hunk 1]
```diff
@@ -15,14 +15,11 @@ const FeaturedProject = (() =>
   <section className="home-featured">
     <h1 className="secondary-kicker">Featured Project</h1>
     <div className="home-featured-images">
-      <img
-        alt="The direct image (left) and dispersed spectrum (right) of a real galaxy from the WISP survey. The white arrow shows the bright light produced by an emission line."
-        src="./assets/featured-projects/featured-project-20170531-galaxy-nurseries.jpg"
-      />
+      <img src="./assets/featured-projects/featured-project-20170711-milky-way.jpg" />
```
Comment (rogerhutchings) [inline app/pages/home-common/featured-project.jsx]: This should be a URL, not a file path (i.e. no `.`)

**Comment by eatyourgreens** [comment]
It's going to need some alt text so it can pass tests and deploy. Maybe Grant can come up with something. It doesn't need to be a lot of text, just a short description of the picture.

**Comment by shaunanoordin** [comment]
Ooh, that's actually a pretty good addition to the tests. OK, will check in with Grant - thanks, Jim!

**Comment by shaunanoordin** [comment]
@rogerhutchings I've changed the `src` URL to `"assets/featured-projects/featured-project-20170711-milky-way.jpg"`; is that what you recommended?

**Comment by rogerhutchings** [comment]
I'd give it an absolute URL path, so beginning with a `/`

**Comment by shaunanoordin** [comment]
👌  Absolute paths are a go

**Pull request merged**